src/libraries/System.Formats.Tar/src/Resources/Strings.resx

```diff
@@ -264,4 +264,7 @@
   <data name="TarEntryFieldExceedsMaxLength" xml:space="preserve">
     <value>The field '{0}' exceeds the maximum allowed length for this format.</value>
   </data>
+  <data name="TarSizeFieldTooLargeForEntryFormat" xml:space="preserve">
+    <value>The value of the size field for the current entry of format '{0}' is beyond the expected length.</value>
```

Comment:
```suggestion
    <value>The value of the size field for the current entry of format '{0}' is greater than the format allows.</value>
```
?

Reply:
tbh, I borrowed the wording from this other string, but I like your suggestion more for this case.
https://github.com/dotnet/runtime/blob/65f9c1c5d5c2c9b2a9d36b1683a2aa2f42c97590/src/libraries/System.Formats.Tar/src/Resources/Strings.resx#L240-L242

Reply:
We can update the other string to match too. I also like the change.

Reply:
Thoughts on changing the other string @carlossanlop?

Reply:
Yeah that sounds fine for me. "Greater than" sounds better. The other one reminds me of Buzz Lightyear's famous words.